# Patch created by Bernard Li <bli@bcgsc.ca> with information provided by Jeremy Siadal
# Intel SATA adapters were incorrectly labelled to use the piix module but instead should be
# using ata_piix

--- discover1-data/pci.lst.orig 2005-09-12 22:39:28.000000000 -0700
+++ discover1-data/pci.lst      2005-07-25 17:30:22.000000000 -0700
@@ -6344,7 +6344,7 @@
 	808624cc	unknown	unknown	82801DBM (ICH4-M) LPC Interface Bridge
 	808624cd	usb	ehci-hcd	82801DB/DBM (ICH4/ICH4-M) USB2 EHCI Controller
 	808624d0	bridge	i810-tco	82801EB/ER (ICH5/ICH5R) LPC Interface Bridge
-	808624d1	ide	piix	82801EB (ICH5) SATA Controller
+	808624d1	ide	ata_piix	82801EB (ICH5) SATA Controller
 	808624d2	usb	usb-uhci	82801EB/ER (ICH5/ICH5R) USB UHCI Controller #1
 	808624d3	unknown	unknown	82801EB/ER (ICH5/ICH5R) SMBus Controller
 	808624d4	usb	usb-uhci	82801EB/ER (ICH5/ICH5R) USB UHCI Controller #2
@@ -6407,7 +6407,7 @@
 	80862592	unknown	unknown	Mobile 915GM/GMS/910GML Express Graphics Controller
 	808625a1	unknown	unknown	6300ESB LPC Interface Controller
 	808625a2	ide	piix	6300ESB PATA Storage Controller
-	808625a3	ide	piix	6300ESB SATA Storage Controller
+	808625a3	ide	ata_piix	6300ESB SATA Storage Controller
 	808625a4	unknown	unknown	6300ESB SMBus Controller
 	808625a6	sound	i810_audio	6300ESB AC'97 Audio Controller
 	808625a7	modem	unknown	6300ESB AC'97 Modem Controller
@@ -6417,7 +6417,7 @@
 	808625ac	unknown	unknown	6300ESB I/O Advanced Programmable Interrupt Controller
 	808625ad	usb	ehci-hcd	6300ESB USB2 Enhanced Host Controller
 	808625ae	unknown	unknown	6300ESB 64-bit PCI-X Bridge
-	808625b0	ide	piix	6300ESB SATA RAID Controller
+	808625b0	ide	ata_piix	6300ESB SATA RAID Controller
 	808625c0	unknown	unknown	Workstation Memory Controller Hub
 	808625d0	unknown	unknown	Server Memory Controller Hub
 	808625d8	unknown	unknown	Server Memory Controller Hub
